help/develop/admin/deploy/deploy-docker-swarm.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Deployment on Docker Swarm"
+---
+
+Datagrok is based on Docker containers, [database](../infrastructure.md#database)
+and [persistent file storage](../infrastructure.md#storage).
+
+As [database](../infrastructure.md#database) Datagrok supports any PostgreSQL database out-of-the-box, including cloud
+solutions for PostgreSQL database, for example [AWS RDS](https://aws.amazon.com/rds/).
+
+For [persistent file storage](../infrastructure.md#storage), Datagrok supports a lot of options, including cloud solutions,
+for example [AWS S3](https://aws.amazon.com/s3/) and Local File System storage.
+
+This document contains instructions to deploy Datagrok using [Docker Swarm](https://docs.docker.com/engine/swarm/)
+on virtual machines or dedicated hosts with Local File System for persistent storage.
+
+More information about Datagrok design and components:
+
+* [Architecture](../architecture.md)
+* [Infrastructure](../infrastructure.md)
+
+## Prerequisites
```

Comment:
Add to prerequisites that it requires 2 machines, one for datagrok and one for cvm. Also, specify the resources as in docker-compose instructions